app/components/hero-section.hbs

```diff
@@ -90,4 +52,41 @@
       />
     </div>
   </section>
```

Comment:
here also why we keep this in ff can you please explain

Reply:
added old one in dev=true will remove after release